www/index.html

```diff
@@ -189,6 +189,7 @@
                 </div>
 
                 <div id="configuration" style="display: none;" class="category-padding">
+                <div id="returntoArticle_top" style="text-align: center;"></div>
```

Comment:
Instead of adding specific links, I would find more user-friendly to return to the previous article when clicking on the Home button (instead of reloading the home page).
It would also work when coming back from the About section.